diff --git a/MCprep_addon/__init__.py b/MCprep_addon/__init__.py
index bf002de..bbd8a23 100755
--- a/MCprep_addon/__init__.py
+++ b/MCprep_addon/__init__.py
@@ -39,13 +39,13 @@ Disclaimer: This is not an official Google product
 # 								error = 51
 
 bl_info = {
-	"name": "MCprep",
+	"name": "MCprep MB",
 	"category": "Object",
-	"version": (3, 6, 1, 3),
+	"version": (3, 6, 1, 9),
 	"blender": (2, 80, 0),
 	"location": "3D window toolshelf > MCprep tab",
 	"description": "Minecraft workflow addon for rendering and animation",
-	"warning": "",
+	"warning": "Not an official release, add MB to the version info in bug reports!",
 	"wiki_url": "https://TheDuckCow.com/MCprep",
 	"author": "Patrick W. Crawford <support@theduckcow.com>",
 	"tracker_url": "https://github.com/TheDuckCow/MCprep/issues"
