diff --git a/MCprep_addon/addon_updater_ops.py b/MCprep_addon/addon_updater_ops.py
index 677b74e..a977dd2 100644
--- a/MCprep_addon/addon_updater_ops.py
+++ b/MCprep_addon/addon_updater_ops.py
@@ -1285,12 +1285,6 @@ def skip_tag_function(self, tag):
 	# 	return True
 	# ---- write any custom code above, return true to disallow version --- #
 	
-	# Ignore release candidates
-	skip_if_present = ['rc', 'alpha', 'beta']
-	for skip_name in skip_if_present:
-		if skip_name in tag.lower():
-			return True
-
 	if self.include_branches:
 		for branch in self.include_branch_list:
 			if tag["name"].lower() == branch:
